protobuf/protoc-gen-govalidator/example/example.proto

```diff
@@ -12,8 +12,8 @@ message ExampleMessage {
   string id = 1 [(validator.uuid) = true];
   // bytes can also be parsed as UUID with support for gogo
   bytes user_id = 2 [(validator.uuid) = true];
-  // strings can validate against a regular expresion
-  string email = 3 [(validator.regex) = "[a-z0-9!#$%&'*+/=?^_{|}~-]+(?:\\.[a-z0-9!#$%&'*+/=?^_{|}~-]+)*@(?:[a-z0-9](?:[a-z0-9-]*[a-z0-9])?\\.)+[a-z0-9](?:[a-z0-9-]*[a-z0-9])?"];
+  // should be a valid email address
+  string email = 3 [(validator.email) = {}];
```

Comment:
@sc-peter @AipengChen 

Is this the correct usage if email is optional? 
```suggestion
  string email = 3 [(validator.email) = { optional = true }];
```

Can we update this line to show the usage of optional?